examples/standalone/runfile/dynamics.py

```diff
@@ -217,12 +221,13 @@ def write_global_timings(experiment, filename, comm, root=0):
 
     # collect times and output simple statistics
     comm.Barrier()
-    if not args.disable_halo_exchange:
-        print("Gathering Times")
-        experiment = set_experiment_info(
-            experiment_name, args.time_step, args.backend, args.hash
-        )
-        gather_timing_statistics(timer, experiment, comm)
+    print("Gathering Times")
+    experiment = set_experiment_info(
+        experiment_name, args.time_step, args.backend, args.hash
+    )
+    gather_timing_statistics(timer, experiment, comm)
+    print(f"{experiment}")
```

Comment:
this should probably no longer be here?

Reply:
Actually I left that there because of the `disable_dump_json`. I figured if you don't dump the full json at least you ought to get a console log of your timings. Which means we need to keep `gather_timing_statistics`